File: app/src/main/java/net/gahfy/functionsextensions/utils/TextViewUtils.java
package net.gahfy.functionsextensions.utils;

import android.content.Context;
import android.graphics.Typeface;
import android.support.annotation.NonNull;
import android.widget.TextView;

public class TextViewUtils {
    public static void setTypefaceToTextView(@NonNull Context context, @NonNull TextView textView, @NonNull String assetPath){
        textView.setTypeface(Typeface.createFromAsset(context.getAssets(), assetPath));
    }
}

File: app/src/main/java/net/gahfy/functionsextensions/utils/TextViewUtils.kt
package net.gahfy.functionsextensions.utils

import android.content.Context
import android.graphics.Typeface
import android.widget.TextView

object TextViewUtils {
    fun setTypefaceToTextView(context: Context, textView: TextView, assetPath: String) {
        textView.typeface = Typeface.createFromAsset(context.assets, assetPath)
    }
}
